Needle.podspec

```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name             = 'Needle'
+  s.version          = '0.8.6'
```

Comment:
The current version is 0.8.6.

Reply:
I was under the assumption that you will bump the version and release it for 0.8.5.  We'd have to update the source path since the directory structure has changed.

Reply:
Good point. I forgot I moved the foundation project out to root to support SPM.

Reply:
Typo: release it for 0.8.6*

Reply:
It seems Needle code generator is not on CocoaPods :( Hopefully you can push the generator too soon. Adding Carthage just for one dependency is not a good option and Using homebrew loses portability of app source code.

Reply:
We don't on having the generator buildable via CocoaPods.  Please use swift package manager to build it from source (if homebrew does not work for you).  Instructions https://github.com/uber/needle/tree/master/Generator

Reply:
AFAIK the code generator as a binary cannot be easily distributed via CocoaPods hence the need for homebrew or building from source.

Reply:
@rudro Alright i see thanks i guess having the binary with the source code would be cleaner than adding another package manager for one dependency, @mzheng I have not seen the binary is in what format but i have seen some frameworks binaries released on pods, on top of my head Google Maps. Also there is an option in pods to compile once from source instead of recompile every time